tests/integration/constraints/test_tabular.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import pandas as pd
+
+from sdv.metadata import SingleTableMetadata
+from sdv.single_table import GaussianCopulaSynthesizer
+
+
+def test_fixed_combinations_with_nans():
```

Comment:
This test is crashing on the `main` branch with the error described in the issue